src/script/cryptography/CryptographyMapper.ts

```diff
@@ -187,12 +191,6 @@ export class CryptographyMapper {
         break;
       }
 
-      case GenericMessageType.IMAGE: {
-        const mappedImage = this._mapImage(genericMessage.image as ImageAsset, event.data.id);
-        specificContent = addMetadata(mappedImage, genericMessage.image);
-        break;
-      }
-
```

Comment:
This is deprecated since 2016, it's long overdue that we remove it :) 
https://github.com/wireapp/generic-message-proto/commit/9d2a5bb8b608b66180af8e1da620651f22fb33ae